podcasts/FolderViewController.swift

```diff
@@ -56,6 +56,8 @@ class FolderViewController: PCViewController, UIGestureRecognizerDelegate {
         mainGrid.addGestureRecognizer(longPressGesture)
         longPressGesture.delegate = self
 
+        miniPlayerStatusDidChange()
```

Comment:
Do we need that here given it's on `viewDidAppear`?

Reply:
Yes or else we see the inset changing animating.